index.js

```diff
@@ -2,6 +2,19 @@
 
 var plugins = require('./plugins.json');
 
+for (const pluginName of Object.keys(plugins)) {
+  let url = plugins[pluginName];
```

Comment:
`let` support is [iffy at best](https://kangax.github.io/compat-table/es6/#test-let) in Node.js v4, keep this as a `var` for now.